prefab_cloud_python/client.py

```diff
@@ -108,7 +108,6 @@ def post(self, path, body):
             "Accept": "application/x-protobuf",
             "Authorization": f"Basic {auth_token}",
         }
-        pool_manager = urllib3.PoolManager(headers=headers)
 
         endpoint = self.options.prefab_api_url.strip("/") + "/" + path.strip("/")
-        pool_manager.request("POST", endpoint, body=body.SerializeToString())
+        requests.post(endpoint, headers=headers, data=body.SerializeToString())
```

Comment:
let's create an instance of `requests.Session()` in the constructor and use that for requests (e.g. `self.session.post()`) so that we reuse the connection pool

Reply:
👍